fixtures/endowedPositions.json

```
@@ -8815,7 +8815,7 @@
   {
     "Key": "KJYCB5289805",
     "SUBCATEGORY": "OFFICE OF THE VICE PROVOST FOR UNDERGRADUATE EDUCATION",
-    "POSITION": "The Margaret Supple College Faculty Fellow",
+    "POSITION": "The Margaret Supple College Faculty Fellow (Srping 2026)",
```

```suggestion
    "POSITION": "The Margaret Supple College Faculty Fellow (Spring 2026)",
```
Looks like a typo? 😄

---

Oy! Yikes. Thanks for checking. Yes, good catch :)

---

I committed it using the button on GitHub - should be rebuilding 😄